7_kyu/Credit_Card_Mask.rb

```diff
@@ -0,0 +1,5 @@
+# https://www.codewars.com/kata/5412509bd436bd33920011bc
+
+def maskify(cc)
+  cc.size > 4 ? cc.last(4).unshift("#" * cc.size - 4) : cc
```

Comment:
First, you should convert to array, for example: ` cc.chars`, or solove without arrray:
```ruby
cc.size > 4 ? cc : '#' * (cc.size-4) + cc[-4..-1]
```
Or use regex solution instead:
```ruby
cc.gsub(/.(?=.{4})/, '#')
```